README.md

```diff
@@ -137,3 +137,33 @@ const browser = await launch();
 // Connect to first browser instead
 const anotherBrowser = await launch({ wsEndpoint: browser.wsEndpoint() });
 ```
+
+## BYOB - Bring Your Own Browser
+
+
+Essentially the process is as simple as running a chromium-like binary with the following flags:
+
+```
+chromium --remote-debugging-port=<SOME_PORT> \
+--headless=new \
+--no-first-run \
+--password-store=basic \
+--use-mock-keychain \
+--hide-scrollbars
+```
+
+> Technically, only the first one is necessary, though I've found that these flags generally get the best result. One this is running, it'd be as simple as
+
+```ts
+// Connect to remote endpoint
+const browser = await launch({
+  wsEndpoint: "wss://localhost:<SOME_PORT>",
```

Comment:
Is wss correct here? I'd be surprised.

Reply:
mm good point, tried an actual run and the launch seems to break on manual WS connection, see examples/byob.ts